specificatie/datum.yaml

```diff
@@ -40,12 +40,16 @@ components:
             datum:
               type: string
               format: date
+            langFormaat:
```

Comment:
je kan ervoor kiezen om deze property toe te voegen aan AbstractDatum. Dan hoeft het niet te worden herhaald voor elke sub type

Reply:
Dan is de Abstract tovh niet meet abstract. Stel dat we ooit een type toe willen voegen waar geeen langFormaat in zit. Of is dit een theoretische discussie ? Ik ben af en toe de weg kwijt tussen zaken zuiver houden of pragmatisch zijn.

Reply:
Abstract betekent niet dat het geen properties mag\kan hebben. In een object geörienteerd programmeer taal betekent het dat je die type niet kan instantieren.
Een reden om te dupliceren is als een sub-type die property niet heeft of als ze niet gelijk zijn, bijv. door het gebruik van een andere pattern voor elke sub-type wat @melsk-r voorstelt.